packages/core/src/credential/Credential.ts

```diff
@@ -6,73 +6,262 @@
  */
 
 /**
- * In KILT, a [[Credential]] is an attested claim, which a Claimer can store locally and share with Verifiers as they wish.
+ * Credentials are a core building block of the KILT SDK.
+ * A Credential represents a [[Claim]] which needs to be validated. In practice, the Credential is sent from a claimer to an attester for attesting and to a verifier for verification.
  *
- * Once a [[RequestForAttestation]] has been made, the [[Attestation]] can be built and the Attester submits it wrapped in a [[Credential]] object.
- * This [[Credential]] also contains the original request for attestation.
- * RequestForAttestation also exposes a [[createPresentation]] method, that can be used by the claimer to hide some specific information from the verifier for more privacy.
+ * A Credential object contains the [[Claim]] and its hash, and legitimations/delegationId of the attester.
+ * It's signed by the claimer, to make it tamper-proof (`claimerSignature` is a property of [[Claim]]).
```

Comment:
> I, the subject of this credential, have seen and agreed to these claims

Wasn’t this an integral part of the flow? Or was it only really needed if a Quote came into play?

Reply:
It was, you're right. But only because we've gotten used to the idea that a claimer signature was somehow required and/or useful on the credential. We figured out at some point that is is really not; for presentation purposes, you want the claimer signature to be "fresh" to prevent replay attacks, so it's only created at that point. And now that the claimer can rotate their keys, this is all the more relevant, because we don't want people to have to search in the history for the key that once signed a credential.

Reply:
Would the signature still be useful for the quote?

Reply:
No, the quote is signed itself and on top of this, so these two don't interact.

Reply:
The only time it is used is when publishing credentials; then you do want to have a signature on it of course. But IMO we may just as well publish a credential presentation instead.
BTW the VC universe also works along these assumptions: here, a credential holder never signs their credential, but signs a credential presentation object, which contains one or more credentials. Abandoning the credential signature thus helps us getting more compliant.

Reply:
My knowledge of the ecosystem is insufficient, but removing the signature from the claim sounds good to me! And I agree about rather doing it sooner than as a separate breaking change.